src/openms/include/OpenMS/METADATA/SpectrumMetaDataLookup.h

```diff
@@ -264,16 +280,30 @@ namespace OpenMS
        @return True if all peptide IDs could be annotated successfully (including if all already had RT values), false otherwise.
 
        Look-up works by matching the "spectrum_reference" (meta value) of a peptide ID to the native ID of a spectrum. Only peptide IDs without RT (where PeptideIdentification::getRT() returns "NaN") are looked up; the RT is set to that of the corresponding spectrum.
-
-       The raw data is only loaded from @p filename if necessary, i.e. if there are any peptide IDs with missing RTs.
     */
-    static bool addMissingRTsToPeptideIDs(
-      std::vector<PeptideIdentification>& peptides, const String& filename,
+    static bool addMissingRTsToPeptideIDs(std::vector<PeptideIdentification>& peptides, const String &filename,
       bool stop_on_error = false);
 
+    /**
+     * @brief Add missing "spectrum_reference"s to peptide identifications based on raw data
+     *
+     * @param peptides Peptide IDs with or without spectrum_reference
+     * @param lookup SpectrumMetaDataLookup with loaded spectra, metadata and optional spectra_data value for Proteins
```

Comment:
for Proteins? " to be annotated in (all I assume???) ProteinIdentification" objects